build.cmd

```diff
@@ -295,6 +296,14 @@ if /i "%ARG%" == "ci_part4" (
     set CI=1
 )
 
+if /i "%ARG%" == "end-2-end" (
+    set BUILD_PROTO=1
+    set BUILD_CORECLR=1
+    set BUILD_NET40_FSHARP_CORE=1
+    set BUILD_NET40=1
+    set TEST_END_2_END=1
```

Comment:
A better name would be ``TEST_TYPEPROVIDER_SAMPLES`` or something.  ``TEST_END_2_END`` is too unspecific

Reply:
Well there will likely be other end-end tests.  But I take your point.